src/config/buttons.js

```diff
@@ -0,0 +1,7 @@
+export const labelsById = {
+  'header-primary-button': 'Watch Video',
+  'header-secondary-button': 'Learn more',
+  carousel: 'Get more info',
```

Comment:
out of curiosity, why are we making some button id's configurable but not others?

Reply:
Actually, none of them are intended to be configurable, but you can't have dashes in the keys without quotation marks.  Maybe I should change to camel case

Reply:
oo, per our goal to tell the client they can feel safe changing all things in quotes in the config file, i think camel case so we can avoid quotes is a great idea!